packages/core/src/electron-main/electron-main-application.ts

```diff
@@ -243,7 +247,15 @@ export class ElectronMainApplication {
         if (browserWindow) {
             this.saveWindowState(browserWindow);
         } else {
-            console.warn(`no BrowserWindow with id: ${webContents.id}`);
+            console.warn(`no BrowserWindow wit  "theia": {
+                "frontend": {
```

Comment:
Why do we need this config info all of a sudden?

Reply:
We don't need it. Reverted.